EpikCord/application.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Optional
+from .utils import TeamMemberMembershipState
+from .flags import Permissions, ApplicationFlags
+from discord_typings import ApplicationData, InstallParams as InstallParamsData, TeamData, TeamMemberData
+
+class InstallParams:
+    def __init__(self, data: InstallParamsData):
+        self.scopes: List[str] = data["scopes"]
+        self.permissions: Permissions = Permissions(int(data["permissions"]))
+
+class TeamMember:
+    def __init__(self, data: TeamMemberData):
+        self.membership_state: TeamMemberMembershipState = TeamMemberMembershipState(data["membership_state"])
```

Comment:
*"TeamMemberMembershipState"*

Reply:
I guess it could be changed to "TeamMembershipState"